CrtanjeLikova/GeometrijskiLikovi/Elipsa.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vsite.CSharp.Metode.GeometrijskiLikovi
+{
+    internal class Elipsa : 
```

Comment:
Niste stavili da je Elipsa izvedena iz klase GeometrijskiLik. Napravite to i onda kliknite desnom tipkom na "Elipsa" - "Quick actions and refactoring" - "Implement abstract class" da Visual Studio generira kosture metoda i da Vam se projekt builda.
Nakon toga pushajte kod i otvorite novi pull request.